roles/post_install/openstack_tasks/tasks/main.yml

```diff
@@ -0,0 +1,97 @@
+---
+
+- name: Setup Openstack environment
+  tags: setup_os_env
+  import_tasks: setup_openstack_env.yml
+  when: setup_os_env
+
+- name: Instances
+  tags: instance
+  import_tasks: instances.yml
+  when: instance and resource_state == 'absent'
+
+- name: Projects, users and roles creation
+  tags: user
+  import_tasks: users.yml
+  when: user and resource_state == 'present'
+
+- name: Quota
+  tags: quota
+  import_tasks: quota.yml
+  when: quota
+
+- name: Volumes
+  tags: volume
+  import_tasks: volumes.yml
+  when: volume
+
+- name: Security groups creation
+  tags: security_group
+  import_tasks: security_groups.yml
```

Comment:
No security_groups.yml either

Reply:
A bunch of files are still not present I will add them one by one. This commit is too big already.

Reply:
Until you add those file, please remove their import from here.

Reply:
Please remove this task until the required file is present